bin/templates/scripts/cordova/lib/build.js

```diff
@@ -87,6 +87,13 @@ module.exports.run = function (argv) {
         }
         return Q.nfcall(fs.writeFile, path.join(__dirname, '..', 'build-extras.xcconfig'), extraConfig, 'utf-8');
     }).then(function () {
+        // Ensure platform scripts are executable as the platform could be added on non-OSX environment
+        shell.find(projectPath).filter(function (file) {
```

Comment:
Makes sense; although there is another script there: 
`ios/build/HelloCordova.build/Debug-iphonesimulator/HelloCordova.build/Script-304B58A110DAC018002A0835.sh`
So we can probably check all folders except `www` - what do you think?